utils.js

```diff
@@ -375,7 +375,8 @@ function formatDeltaMessage(m){
     messageID: md.messageId,
     attachments: (m.delta.attachments || []).map(v => _formatAttachment(v)),
     timestamp: md.timestamp,
-    isGroup: !!md.threadKey.threadFbId
+    isGroup: !!md.threadKey.threadFbId,
+    reactions: originalMessage.reactions ? originalMessage.reactions : []
```

Comment:
Well, when I thinking about this, the reactions in deltaMessage is not required because all reactions communication going through ClientPayload